packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -493,261 +489,303 @@ export class StatsAnalyzer extends EventsScope {
    */
   private compareLastStatsResult() {
     if (this.lastStatsResults !== null && this.meetingMediaStatus) {
-      // compare audio stats sent
-      let currentStats = null;
-      let previousStats = null;
-
-      Object.keys(this.statsResults).forEach((mediaType) => {
-        if (!this.lastStatsResults[mediaType]) {
-          return;
-        }
-
-        if (this.meetingMediaStatus.expected.sendAudio && mediaType.includes('audio-send')) {
-          currentStats = this.statsResults[mediaType].send;
-          previousStats = this.lastStatsResults[mediaType].send;
-
+      const getCurrentStatsTotals = (keyPrefix: string, value: string): number =>
+        Object.keys(this.statsResults)
+          .filter((keys) => keys.includes(keyPrefix))
+          .reduce((prev, cur) => prev + (this.statsResults[cur].recv[value] || 0), 0);
+
+      const getPreviousStatsTotals = (keyPrefix: string, value: string): number =>
+        Object.keys(this.statsResults)
+          .filter((keys) => keys.includes(keyPrefix))
+          .reduce((prev, cur) => prev + (this.lastStatsResults[cur].recv[value] || 0), 0);
+
+      const getCurrentResolutionsStatsTotals = (keyPrefix: string, value: string): number =>
```

Comment:
this function is exactly the same like `getCurrentStatsTotals()` or am I missing something?
same for `getPreviousResolutionsStatsTotals()` being same as `getPreviousStatsTotals()`

Reply:
Yes this was missing `.resolutions`